### DISCUSSION TITLE
Welcome to TorchHeightController Discussions!

### DISCUSSION THREAD

**Discussion by HaleDesign** [HaleDesign/TorchHeightController#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mikeyd2673** [comment]
Hello, im wondering if anyone has or had the same issue im having and could possibly help me figure it out.
I'm right now trying to build this torch height control for my home made cnc plasma cutter.
I cant get it to work I have tried the uno and the mega 
The screen is there and it is talking to the arduino it reads the plasma voltage simulated by a potentiometer the set point reads 119.14 and the plasma voltage changes when I play with the potentiometer 
But my current position stays at zero no matter if I turn the potentiometer all the way up or down. The current position never moves 
Can you please help me to figure out were im going wrong please 
Thank you for your time

**Reply by Anthony-Andrews** [reply]
Hello, I'm also having the exact same issue - "the current position stays at zero no matter if I turn the potentiometer up or down."
I've tried the sketch from March 1, 2021 as well as the sketch from Dec. 30, 2020 to no avail.  Has anyone found a solution to this problem?
Thank you!

**Reply by SmokyMtnRed** [reply]
I was on the edge and ready to jump into doing this setup, but waiting to see if this issue is fixed. Has anyone found a solution?

**Reply by PromachNZ** [reply]
im trying at the moment shouldn't be a big problem.

**Comment by gys417** [comment]
can i use another nextion screen the one specify not availible in south africa

**Reply by sapler2** [reply]
i transform nextion project from 3,5" display to 2,4" but have troubles with button reactions in settings menu, somebody have similiar issues ?
